Cesium3DTilesSelection/src/TileContentFactory.cpp

```diff
@@ -35,6 +35,27 @@ void TileContentFactory::registerContentType(
 
 CesiumAsync::Future<std::unique_ptr<TileContentLoadResult>>
 TileContentFactory::createContent(const TileContentLoadInput& input) {
+  // This is a hack to support gltf/glb content directly. Is there a more
+  // elegant way to do this instead of check the extension here?
+  const std::string& url = input.pRequest->url();
+  size_t extensionPos = url.rfind(".");
```

Comment:
It should now be removing the query parameters. Is there anything else I should look for and cut off on the rhs of the url before I look for the extension? Are hash parameters  possible in this situation? I assumed not.

Reply:
No I think this is fine. Ideally we'd use `uriparser` instead of a manual URI manipulation, but it's ok for now. Really we need a better URI library that isn't such a hassle to use (mostly cause it's 'C' and very low-level).